penquins/penquins.py

```diff
@@ -286,3 +318,97 @@ def ping(self) -> bool:
                 print(_e)
                 print(_err)
             return False
+
+    def query_skymap(
+        self,
+        path: Path,
+        cumprob: float,
+        start_date: str,
+        end_date: str,
+        min_detections: int,
+        catalogs: List[str],
+        program_id: int,
+        n_treads=6,
+    ) -> List[dict]:
+        missing_args = [
+            arg
+            for arg in [
+                path,
+                cumprob,
+                start_date,
+                end_date,
+                min_detections,
+                catalogs,
+                program_id,
+            ]
+            if arg is None
+        ]
+        if len(missing_args) > 0:
+            raise ValueError(f"Missing arguments: {missing_args}")
+
+        cones = get_cones(path, cumprob)
+        jd_start = Time(start_date).jd
+        jd_end = Time(end_date).jd
+
+        queries = []
```

Comment:
Is this similar to the code inside skyportal that also does this check?

Reply:
I changed that since then. Now I take jd dates as an input.